src/VASSAL/build/module/Chatter.java

```diff
@@ -21,477 +21,636 @@
-      @Override
+		gameMsg3Color.addPropertyChangeListener(new PropertyChangeListener() {
+			@Override
       public void propertyChange(PropertyChangeEvent e) {
-        myChat = (Color) e.getNewValue();
-      }
-    });
+				gameMsg3 = (Color) e.getNewValue();
+				makeStyleSheet(null);
+			}
+		});
+
+		globalPrefs.addOption(Resources.getString("Chatter.chat_window"), gameMsg3Color);
+
+		gameMsg3 = (Color) globalPrefs.getValue(GAME_MSG3_COLOR);
+		
+		
+		//BR//
+		//BR// game message color #4 (messages starting with "~")
+		//BR//
+		final ColorConfigurer gameMsg4Color = new ColorConfigurer(GAME_MSG4_COLOR,
+				Resources.getString("Chatter.game_messages_preference") + "(#4 - first character \"~\"):  ", new Color(255, 0, 0));
+
+		gameMsg4Color.addPropertyChangeListener(new PropertyChangeListener() {
+			@Override
+      public void propertyChange(PropertyChangeEvent e) {
+				gameMsg4 = (Color) e.getNewValue();
+				makeStyleSheet(null);
+			}
+		});
+
+		globalPrefs.addOption(Resources.getString("Chatter.chat_window"), gameMsg4Color);
+
+		gameMsg4 = (Color) globalPrefs.getValue(GAME_MSG4_COLOR);
+		
+		
+		//BR//
+		//BR// game message color #5 (messages starting with "`")
+		//BR//
+		final ColorConfigurer gameMsg5Color = new ColorConfigurer(GAME_MSG5_COLOR,
+				Resources.getString("Chatter.game_messages_preference") + "(#5 - first character \"`\"):  ", new Color(153, 0, 153));
+
+		gameMsg5Color.addPropertyChangeListener(new PropertyChangeListener() {
+			@Override
+      public void propertyChange(PropertyChangeEvent e) {
+				gameMsg5 = (Color) e.getNewValue();
+				makeStyleSheet(null);
+			}
+		});
+
+		globalPrefs.addOption(Resources.getString("Chatter.chat_window"), gameMsg5Color);
+
+		gameMsg5 = (Color) globalPrefs.getValue(GAME_MSG5_COLOR);
 
-    globalPrefs.addOption(
-      Resources.getString("Chatter.chat_window"), myChatColor
-    );
 
-    myChat = (Color) globalPrefs.getValue(MY_CHAT_COLOR);
 
-    //
-    // other message color
-    //
-    final ColorConfigurer otherChatColor = new ColorConfigurer(
-      OTHER_CHAT_COLOR,
-      Resources.getString("Chatter.other_text_preference"),
-      Color.black
-    );
+		//
+		// system message color
+		//
+		final ColorConfigurer systemMsgColor = new ColorConfigurer(SYS_MSG_COLOR,
+				Resources.getString("Chatter.system_message_preference"), new Color(160, 160, 160));
 
-    otherChatColor.addPropertyChangeListener(new PropertyChangeListener() {
+		systemMsgColor.addPropertyChangeListener(new PropertyChangeListener() {
+			@Override
+      public void propertyChange(PropertyChangeEvent e) {
+				systemMsg = (Color) e.getNewValue();
+				makeStyleSheet(null);
+			}
+		});
+
+		globalPrefs.addOption(Resources.getString("Chatter.chat_window"), systemMsgColor);
+
+		systemMsg = (Color) globalPrefs.getValue(SYS_MSG_COLOR);
+
+
+		//
+		// my message color
+		//
+		final ColorConfigurer myChatColor = new ColorConfigurer( MY_CHAT_COLOR,
```

Comment:
Can name the variable `myMessageColor` and remove the comment above

Reply:
fixed for next push